playbook/app/pb_kits/playbook/pb_form_pill/_form_pill.scss

```diff
@@ -51,20 +58,18 @@ $form_pill_colors: (
         @if ($color_name == "neutral") {
           color: $text_lt_default;
         }
-        padding-left: $space-sm;
-        padding-right: calc($space-sm/2);
+        padding: 0 $space-xs;
       }
       #{$selector}_close {
         color: $color_value;
-        padding-left: calc($space-sm/4);
-        padding-right: calc($space-sm/4);
         display: flex;
         align-items: center;
-        // I had to temporarily change height to 27px so new hover state darker background forms a circle not an oval
-        // before size change (ticket 2 of 4) - change back to 100% when $pb_form_pill_height changed to 27px from 37px
-        height: 27px;
-        border-radius: 70px;
+        height: 17px;
+        border-radius: calc(50%);
```

Comment:
This can just be `50%`

Reply:
Unfortunately 50 % on its own did not give what we wanted - experimented with that and several other attempts while pairing with a designer last week (there was a stack overflow where someone else also noted a difference in a similar thing). I'd also prefer just 50 % and am not sure why they calculate differently but this clunky-construction got the design OK.

Reply:
Strange! Good to know, though.